### PR TITLE
Add PeerReport Header

### LINKED ISSUE #4385: Add documentation to the details of connected peers
We don't have it in docs:
peers

### PR DESCRIPTION
**Pull request by smartprogrammer93**
Resolves #4385

## Changes:
Added Header to Peer Report

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe): 

## Testing
**Requires testing**

- [x] Yes
- [ ] No

**In case you checked yes, did you write tests??**

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by LukaszRozmej** [comment]
Isn't that a bit lacking? While those are categories it doesn't describe particular columns.

**Comment by smartprogrammer93** [comment]
> Isn't that a bit lacking? While those are categories it doesn't describe particular columns.

@LukaszRozmej I agree but I do not have enough space to put a header for all columns. so i thought something is better than nothing

**Comment by LukaszRozmej** [comment]
Maybe just names so for bandwidth you can put H/B/R/S for Headers, Blocks, Receipts and State. It might not be that self-explanatory, but not cryptic either, especially we already mark allocations like that in the first column.

File: src/Nethermind/Nethermind.Synchronization/Peers/SyncPeersReport.cs [hunk 1]
```diff
@@ -62,7 +62,8 @@ public void WriteFullReport()
                 }
 
                 RememberState(out bool _);
-                _stringBuilder.Append($"Sync peers - Initialized: {_currentInitializedPeerCount} | All: {_peerPool.PeerCount} | Max: {_peerPool.PeerMaxCount}");
+                _stringBuilder.AppendLine($"Sync peers - Initialized: {_currentInitializedPeerCount} | All: {_peerPool.PeerCount} | Max: {_peerPool.PeerMaxCount}");
+                AddPeerHeader();
```
Comment (marcindsobczak) [inline src/Nethermind/Nethermind.Synchronization/Peers/SyncPeersReport.cs:66]: We sometimes have headers descriptions without any data below. Maybe let's print headers only if OrderedPeers is not empty?

header

Comment (marcindsobczak) [reply src/Nethermind/Nethermind.Synchronization/Peers/SyncPeersReport.cs:66]: @smartprogrammer93

**Pull request merged**